list/List.js

```diff
@@ -182,6 +182,8 @@ define([
 
 		template: template,
 
+		tabindex: 0,
```

Comment:
The property is actually `tabIndex` with a capital I.  In theory you should be able to add that property and then it works, but you'll have to see if you get problems on Chrome.